modules/destroy-hc-ibmz-cli.adoc

```diff
@@ -17,6 +17,12 @@ To destroy a hosted cluster and its managed cluster on `x86` bare metal with {ib
 $ oc -n <hosted_cluster_namespace> scale nodepool <nodepool_name> --replicas 0
 ----
 +
+After the `NodePool` is scaled to `0`, the compute nodes get detached from the hosted cluster. In {product-title} version 4.17, this function is applicable only for {ibm-z-title} with KVM. For z/VM and LPAR we must delete the compute nodes manually.
++
+If you want to re-attach compute nodes to the cluster, you can scale down a nodepool with desired number of compute nodes. Unlike {ibm-z-title} with KVM, in case of z/VM and LPAR  to reuse the agents ,you must recreate them by using the Discovery image.
+
+When you scale down a node pool, Agents are unbound from the corresponding cluster. Before you can reuse the Agents, you must restart them by using the Discovery image.
```

Comment:
this following statement can be removed from here, as we explained the scenario specifically in the above statements

_When you scale down a node pool, Agents are unbound from the corresponding cluster. Before you can reuse the Agents, you must restart them by using the Discovery image._

Reply:
Done